core/src/test/java/jenkins/model/JDKNameTest.java

```diff
@@ -0,0 +1,59 @@
+/*
+ * The MIT License
+ *
+ * Copyright (c) 2014
```

Comment:
Probably. BTW it's up to the commit author. Probably he developed this code in 2014